### PR TITLE
fix typos in misc constructs in regex

### PR DESCRIPTION
**Pull request by herohua**
## Summary

Fix typos in misc constructs in regex.

### CODE REVIEW

File: docs/standard/base-types/miscellaneous-constructs-in-regular-expressions.md [hunk 1]
```diff
@@ -57,16 +57,16 @@ Regular expressions in .NET include three miscellaneous language constructs. One
...
   
- The following example repeats the first regular expression pattern from the example in the previous section. It adds two inline comments to the regular expression to indicate whether the comparison is case-sensitive. The regular expression pattern, `\b((?# case-sensitive comparison)D\w+)\s((?#case-insensitive comparison)d\w+)\b`, is defined as follows.  
+ The following example repeats the first regular expression pattern from the example in the previous section. It adds two inline comments to the regular expression to indicate whether the comparison is case-sensitive. The regular expression pattern, `\b((?# case sensitive comparison)D\w+)\s(?ixn)((?#case insensitive comparison)d\w+)\b`, is defined as follows.  
   
 |Pattern|Description|  
 |-------------|-----------------|  
 |`\b`|Start at a word boundary.|  
-|`(?# case-sensitive comparison)`|A comment. It does not affect pattern-matching behavior.|
```
Comment (rpetrusha) [inline docs/standard/base-types/miscellaneous-constructs-in-regular-expressions.md]: The original line is correct. The comment in the regex (and in the element-by-element description of the regex pattern) should have been "case-sensitive comparison".

File: docs/standard/base-types/miscellaneous-constructs-in-regular-expressions.md [hunk 2]
```diff
@@ -57,16 +57,16 @@ Regular expressions in .NET include three miscellaneous language constructs. One
...
   
 |Pattern|Description|  
 |-------------|-----------------|  
 |`\b`|Start at a word boundary.|  
-|`(?# case-sensitive comparison)`|A comment. It does not affect pattern-matching behavior.|  
+|`(?# case sensitive comparison)`|A comment. It does not affect pattern-matching behavior.|  
 |`(D\w+)`|Match a capital "D" followed by one or more word characters. This is the first capturing group.|  
 |`\s`|Match a white-space character.|  
 |`(?ixn)`|From this point on, make comparisons case-insensitive, make only explicit captures, and ignore white space in the regular expression pattern.|  
-|`(?#case-insensitive comparison)`|A comment. It does not affect pattern-matching behavior.|
```
Comment (rpetrusha) [inline docs/standard/base-types/miscellaneous-constructs-in-regular-expressions.md]: The original line is correct. The comment in the regex (and in the element-by-element description of the regex pattern) should have been "case-insensitive comparison".

**Review by rpetrusha — CHANGES_REQUESTED** [review]
Thanks for continuing to correct additional errors, omissions, and typos in the regex documentation, @herohua. Here, though, case-sensitive and case-insensitive (with a hyphen) is correct. Instead of changing the text here, the source code at samples/snippets/csharp/VS_Snippets_CLR/regularexpressions.language.miscellaneous/cs/miscellaneous2.cs#2 and samples/snippets/visualbasic/VS_Snippets_CLR/regularexpressions.language.miscellaneous/vb/miscellaneous2.vb#2 should be modified to include a hyphen after "case".    
Please let us know whether you'd like to make that change, or whether you'd like for us to make it.

**Comment by herohua** [comment]
@rpetrusha , understood. I will update the PR accordingly and can help with changes in source code after this PR.

**Review by rpetrusha — APPROVED** [review]
Thanks, @herohua, for the additional commit. I'll merge your PR now. The changes should be live on docs.microsoft.com in the next day or two.

**Pull request merged**